lib/defender/aws_iot_defender.c

```diff
@@ -74,6 +79,8 @@ static void _guardTimerExpirationRoutine( void * );
 
 uint32_t _AwsIotDefenderMetricsFlag[ _DEFENDER_METRICS_GROUP_COUNT ];
 
+AwsIotMutex_t _AwsIotDefenderMetricsMutex;
```

Comment:
How about placing all of this global types in a struct? Would it increase readability?

Reply:
make sense.  Let me change it.